packages/experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/ExtendedSelectedItem.tsx

```diff
@@ -0,0 +1,128 @@
+/* tslint:disable */
+import * as React from 'react';
+/* tslint:enable */
+import { BaseComponent, autobind, css, getId } from '../../../../Utilities';
+import { Persona, PersonaSize, PersonaPresence } from 'office-ui-fabric-react/lib/Persona';
+import { ValidationState } from 'office-ui-fabric-react/lib/Pickers';
+import { IExtendedPersonaProps, ISelectedPeopleItemProps } from '../SelectedPeopleList';
+import { IconButton } from 'office-ui-fabric-react/lib/Button';
+import { ContextualMenu, DirectionalHint, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+import * as stylesImport from './ExtendedSelectedItem.scss';
+// tslint:disable-next-line:no-any
+const styles: any = stylesImport;
+
+export interface IPeoplePickerItemState {
+  contextualMenuVisible: boolean;
+}
+
+export class ExtendedSelectedItem extends BaseComponent<ISelectedPeopleItemProps, IPeoplePickerItemState> {
+  protected persona: HTMLElement;
+
+  private menuItems: IContextualMenuItem[];
+
+  constructor(props: ISelectedPeopleItemProps) {
+    super(props);
+    this.state = { contextualMenuVisible: false };
+    let { onCopyItem, onRemoveItem, copyMenuItemText, removeMenuItemText } = this.props;
+    this.menuItems = [
```

Comment:
Why not let the consumer pass in other menu items?

Reply:
I'm actually pretty sure someone will want to pass in their own menu items.

Reply:
You're right. I exposed the menuItems as a prop, so they can be passed in